src/components/withKeyboardState.tsx

```diff
@@ -52,11 +55,9 @@ export default function withKeyboardState(WrappedComponent) {
                     ref={ref}
                 />
             )}
-        </KeyboardStateContext.Consumer>
-    ));
-
-    WithKeyboardState.displayName = `withKeyboardState(${getComponentDisplayName(WrappedComponent)})`;
+        </KeyboardStateContext.Consumer>));
+    (WithKeyboardState as unknown as {displayName: string}).displayName = `withKeyboardState(${getComponentDisplayName(WrappedComponent as ComponentType)})`;
```

Comment:
@fvlvte You can add a type to `WithKeyboardState` above so you can get rid of type assertions here. 
Also, have you tested this HOC type? maybe create a simple TS component, wrap it with HOC, and see if typing is ok.